corehq/apps/app_manager/models.py

```diff
@@ -749,6 +749,7 @@ class FormBase(DocumentSchema):
     no_vellum = BooleanProperty(default=False)
     form_links = SchemaListProperty(FormLink)
     schedule_form_id = StringProperty()
+    case_references = ListProperty()
```

Comment:
A `SchemaListProperty` would make it clear what data this list contains. Will the same list be used once Vellum supports saving case properties? Could be good to think of the end design now rather than having to refactor a bunch later.